api/config/module/sourcecontrol/v1/sourcecontrol.proto

```diff
@@ -0,0 +1,15 @@
+syntax = "proto3";
+
+package clutch.config.module.sourcecontrol.v1;
+
+option go_package = "github.com/lyft/clutch/backend/api/config/module/sourcecontrol/v1;sourcecontrolv1";
+
+import "sourcecontrol/v1/sourcecontrol.proto";
+
+message Config {
+  // Name of the owner for new repositories.
+  repeated string owners = 1;
```

Comment:
Does owners here imply organizations at least for a github context?

Reply:
It can be both organizations as well as a member since you can create accounts for the organizations you belong to but also your own github account.